instrumentation/opentelemetry-instrumentation-redis/tests/test_redis.py

```diff
@@ -148,6 +148,45 @@ def request_hook(span, conn, args, kwargs):
         span = spans[0]
         self.assertEqual(span.attributes.get(custom_attribute_name), "GET")
 
+    def test_query_sanitizer_enabled(self):
+        redis_client = redis.Redis()
+        connection = redis.connection.Connection()
+        redis_client.connection = connection
+
+        RedisInstrumentor().uninstrument()
+        RedisInstrumentor().instrument(
+            tracer_provider=self.tracer_provider
+        )
```

Comment:
I think you don't need this part, it is already instrumented properly

Reply:
I've removed the unnecessary setup code here.